charts/alfresco-transform-service/templates/deployment-transform-router.yaml

```diff
@@ -53,25 +57,41 @@ spec:
           imagePullPolicy: {{ .Values.transformrouter.image.pullPolicy }}
           {{- include "component-security-context" .Values.transformrouter | indent 8 }}
           envFrom:
-          - configMapRef:
-              name: {{ template "alfresco-transform-service.config-transform-router.name" . }}
-          - secretRef:
-              name: {{ default (printf "%s-messagebroker-secret" (include "alfresco-transform-service.fullname" $)) $.Values.messageBroker.existingSecretName }}
+            - configMapRef:
+                name: {{ template "alfresco-transform-service.config-transform-router.name" . }}
           env:
-            {{- include "activemq.env" . | nindent 12 }}
+            {{- include "alfresco-transform-service.messageBroker.env" $ | nindent 12 }}
           ports:
             - containerPort: {{ .Values.transformrouter.image.internalPort }}
           resources: {{- toYaml .Values.transformrouter.resources | nindent 12 }}
           {{ include "alfresco-transform-service.probes" .Values.transformrouter | nindent 10 }}
-          {{- if .Values.ai.enabled }}
           volumeMounts:
-          - name: config-volume
-            mountPath: /mnt/routes
+          {{- if .Values.ai.enabled }}
+            - name: config-volume
+              mountPath: /mnt/routes
+          {{- end }}
```

Comment:
You can remove that as I think it's better to deal with it from the umbrella chart as a generic extra volumeMount